src/Command/DumpEncryptedFieldsMapCommand.php

```diff
@@ -63,24 +60,25 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $dumper = new Dumper();
 
         foreach ($this->documentManagers as $name => $documentManager) {
-            $generator          = new EncryptedFieldsMapGenerator($documentManager->getMetadataFactory());
-            $encryptedFieldsMap = $generator->getEncryptedFieldsMap();
+            $encryptedFieldsMap = [];
+            foreach ($documentManager->getMetadataFactory()->getAllMetadata() as $metadata) {
+                $database               =  $documentManager->getDocumentDatabase($metadata->getName());
+                $collectionInfoIterator = $database->listCollections(['filter' => ['name' => $metadata->getCollection()]]);
+
+                foreach ($collectionInfoIterator as $collectionInfo) {
+                    if ($collectionInfo['options']['encryptedFields'] ?? null) {
+                        $encryptedFieldsMap[$this->getDocumentNamespace($metadata, $database->getDatabaseName())] = $collectionInfo['options']['encryptedFields'];
```

Comment:
Do we want to merge encrypted fields maps if one already exists? This could happen in inheritance scenarios. Not sure if we want to consider the use case and its implications, but it wouldn't be too hard to ensure that all fields are encrypted.